src/OpenTelemetry.ResourceDetectors.ProcessRuntime/README.md

```diff
@@ -0,0 +1,42 @@
+# Process Runtime Resource Detectors
+
+[![NuGet version badge](https://img.shields.io/nuget/v/OpenTelemetry.ResourceDetectors.ProcessRuntime)](https://www.nuget.org/packages/OpenTelemetry.ResourceDetectors.ProcessRuntime)
+[![NuGet download count badge](https://img.shields.io/nuget/dt/OpenTelemetry.OpenTelemetry.ResourceDetectors.ProcessRuntime)](https://www.nuget.org/packages/OpenTelemetry.ResourceDetectors.ProcessRuntime)
+
+## Getting Started
+
+You need to install the
+`OpenTelemetry.ResourceDetectors.ProcessRuntime` to be able to use the
+Process Runtime Resource Detectors. It detects `process.runtime.description`,
+`process.runtime.name`, and `process.runtime.version` from .NET runtime.
+
+```shell
+dotnet add package OpenTelemetry.ResourceDetectors.ProcessRuntime --prerelease
+```
+
+## Usage
+
+You can configure Process Runtime resource detector to
+the `TracerProvider` with the following example below.
+
+```csharp
+using OpenTelemetry;
+using OpenTelemetry.ResourceDetectors.ProcessRuntime;
+
+var tracerProvider = Sdk.CreateTracerProviderBuilder()
+                        // other configurations
+                        .SetResourceBuilder(ResourceBuilder
+                            .CreateEmpty()
+                            .AddDetector(new ProcessRuntimeDetector()))
+                        .Build();
+```
+
+The resource detectors will record the following metadata based on where
```

Comment:
The same information is also mentioned in the Getting Started section of this doc. We could mention this only once.

Reply:
Fixed 867ad08faa3d5ebcedf2078dd7098a8635a0ca15